watcher/standard/src/main/java/io/fabric8/maven/watcher/standard/SpringBootWatcher.java

```diff
@@ -70,8 +80,46 @@ public void watch(List<ImageConfiguration> configs, Set<HasMetadata> resources,
 
         new PodLogService(logContext).tailAppPodsLogs(kubernetes, getContext().getNamespace(), resources, false, null, true, null, false);
 
+        String url = getServiceExposeUrl(kubernetes, resources);
+        if (url == null) {
+            url = getPortForwardUrl(resources);
```

Comment:
[2] One of my highlights of this PR, to work without a service which also requires extra tools like `exposecontroller` installed. Thanks a lot for this ;-). I wonder whether we should not get rid of the whole service logic and only use the pod port forwarding, since this works with much less dependencies. 

wdyt ? this would also reduce the bug surface quite a bit.